src/ucp/core/ucp_worker.c

```diff
@@ -666,11 +643,6 @@ static void ucp_worker_close_ifaces(ucp_worker_h worker)
             }
         }
 
-        if (ucp_worker_is_tl_tag_offload(worker, rsc_index)) {
```

Comment:
if ucp_worker_is_tl_tag_offload() isn't called from anywhere anymore, can remove this function.